### DISCUSSION TITLE
Windows 11 xmgrace

### DISCUSSION THREAD

**Discussion by Kelvin-Ngowi** [nrc-cnrc/EGSnrc#1038] [Questions]
can anyone please tell me how can I run xmgrace on windows 11

**Comment by karl63** [comment]
As Xmgrace is not specific to egsnrc. I would  use google to find an answer to questions like this one of yours. You should try this too !!! When I Search for Xmgrace and Windows, I find , that the website at Weizman , where XMgrace is maintained says that xmgrace is for unix - like systems so not for windows. I find however also that some people  reccomend qtgrace for windows. I'm working with linux, therefore I never worked with qtgrace. Give it a try! I also found, that xmgrace is on cygwin, too.    You could try this as well
